staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -136,7 +136,7 @@ func NewController(netIfc kosclientv1a1.NetworkV1alpha1Interface,
 			Subsystem: metricsSubsystem,
 			Name:      "attachment_create_to_lock_latency_seconds",
 			Help:      "Latency from Attachment CreationTimestamp to IPLock CreationTimestamp, in seconds",
-			Buckets:   []float64{-1, 0, 1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 64},
+			Buckets:   []float64{-1, 0, 0.125, 0.25, 0.5, 1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 64},
```

Comment:
Subsecond resolution will not help here because the values observed are in whole seconds.  These values are differences in ObjectMeta.CreationTimestamp values, and those are in whole seconds.

You might consider taking readings from the new metadata instead.

Reply:
You correctly identified the problem and the solution.
1cf5b82 switches to using the extended metadata.